lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -47,7 +47,7 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
       <option value="Errors/Technical Issues">{gettext('Errors/Technical Issues')}</option>
       <option value="Financial Aid">{gettext('Financial Aid')}</option>
       <option value="Masters">{gettext('Masters')}</option>
-      <option value="MicroBachelors">{gettext('MicroBachelors')}</option>
+      <option value="Micro">{gettext('Micro')}</option>
```

Comment:
Was the ticket asking for _MicroBachelors_ to be changed to _Micro_, or for _MicroMasters_ to be added as an option?